src/sql/parts/notebook/cellViews/textCell.component.ts

```diff
@@ -137,7 +137,7 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 
 	public toggleEditMode(editMode?: boolean): void {
 		this.isEditMode = editMode !== undefined? editMode : !this.isEditMode;
-		this.updatePreviewCssClass();
+
```

Comment:
Why is this no longer needed? I'm a little unclear on this part of the change

Reply:
@kevcunnane , confirmed with Rony, we don't need to have two styles for editing and preview. So removed codes not used.